source/brailleTables.py

```diff
@@ -148,6 +149,9 @@ def listTables():
 addTable("en-us-comp8.ctb", _("English (U.S.) 8 dot computer braille"))
 # Translators: The name of a braille table displayed in the
 # braille settings dialog.
+addTable("en-us-comp8-ext.utb", _("English (U.S.) 8 dot extended computer braille"))

```

Comment:
I wonder if we can (and want to) drop en-us-comp8.ctb now:
1. Because it is a subset of the 'extended' table. The only reason not to do this is that users prefer the hexadecimal codes over some dot patterns they may not be familiar with. However, I think this is unlikely since most dot patterns match those in other major screen readers for Windows.
2. Because it is a bit confusing which table does what.

Arguably I should bring this up with the liblouis team, but nothing will happen on that front before NVDA 2017.3.